mbzirc_ign/models/mbzirc_oberon7_arm/model.sdf.erb

```diff
@@ -395,9 +395,44 @@ end
           </mesh>
         </geometry>
       </visual>
+
+      <visual name="camera_visual">
+        <pose>0.19 0.075 0.0 -1.570796326794896 0 0</pose>
```

Comment:
Maybe this could be a variable? It's also used in the `<sensor><pose>` block.

Reply:
ok yeah added var in [53541ab](https://github.com/osrf/mbzirc/pull/75/commits/53541aba6d20e6bbf7259182228ceeed29c83098)